code/utility/src/palindrome/palindrome_check/palindrome_check.py

```diff
@@ -0,0 +1,16 @@
+def isPalindrome(str):
+ 
+    # Run loop from 0 to len/2 
+    for i in xrange(0, len(str)/2): 
```

Comment:
Python2 is not allowed.
Use `range` instead of `xrange`.
Use `//` instead of `/` for divide.